src/CommonITILValidation.php

```diff
@@ -318,6 +318,7 @@ public function post_addItem()
             if (
                 ($item->fields['global_validation'] == self::ACCEPTED)
                 || ($item->fields['global_validation'] == self::NONE)
+                || ($item->fields['global_validation'] == self::REFUSED)
             ) {
```

Comment:
IMHO, the only case that should not trigger a computation is when the status is already `WAITING`.
```suggestion
            if (((int) $item->fields['global_validation']) !== self::WAITING) {
```